docs/src/main/sphinx/release/release-406.md

```diff
@@ -0,0 +1,96 @@
+# Release 406 (18 Jan 2023)
+
+## General
+
+* Add support for [exchange spooling on HDFS](fte-exchange-hdfs) when
+  fault-tolerant execution is enabled. ({issue}`15160`)
+* Improve planner estimates for queries containing outer joins over a subquery
+  involving `ORDER BY` and `LIMIT`. ({issue}`15428`)
+* Improve accuracy memory usage reporting for table scans. ({issue}`15711`)
+* Improve performance of queries parsing date values in ISO 8861 format. ({issue}`15548`)
+* Remove `legacy-phased` execution scheduler. ({issue}`15657`)
+* Fix failure when `WHERE` or `JOIN` clauses contain a `LIKE` expression with a
+  non-constant pattern or escape. ({issue}`15629`)
+* Fix inaccurate planner estimates for queries with filters on columns without
+  statistics. ({issue}`15642`)
+* Fix queries with outer joins failing when fault-tolerant execution is
+  enabled. ({issue}`15608`)
+* Fix potential query failure when using `MATCH_RECOGNIZE`. ({issue}`15461`)
+* Fix query failure when using group-based access control with column masks or
+  row filters. ({issue}`15583`)
+* Fix potential hang during shutdown. ({issue}`15675`)
+
+## BigQuery connector
+
+* Fix possible incorrect results for certain queries like `count(*)` when a
+  table has recently been written to. ({issue}`14981`)
+
+## Delta Lake connector
+
+* Improve performance of reading Parquet files. ({issue}`15498`)
+* Improve memory accounting of the Parquet reader. ({issue}`15554`)
+* Improve performance of queries with filters or projections on low-cardinality
+  string columns stored in Parquet files. ({issue}`15269`)
+* Fix reading more data than necessary from Parquet files for queries with
+  filters. ({issue}`15552`)
+* Fix potential query failure when writing to Parquet from a table with an
+  `INTEGER` range on a `BIGINT` column. ({issue}`15496`)
+* Fix query failure due to missing null counts in Parquet column indexes. ({issue}`15706`)
+
+## Hive connector
+
+* Add support for table redirections to catalogs using the Hudi connector. ({issue}`14750`)
+* Improve performance of reading Parquet files. ({issue}`15498`)
+* Improve memory accounting of the Parquet reader. ({issue}`15554`)
+* Improve performance of queries with filters or projections on low-cardinality
+  string columns stored in Parquet files. ({issue}`15269`)
+* Improve performance of queries with filters when Bloom filter indexes are
+  present in Parquet files. Use of Bloom filters from Parquet files can be
+  disabled with the `parquet.use-bloom-filter` configuration property or the
+  `parquet_use_bloom_filter` session property. ({issue}`14428`)
+* Allow coercion between `UNIONTYPE` and `STRUCT`-typed columns. ({issue}`15017`)
```

Comment:
```suggestion
* Allow coercion between Hive `UNIONTYPE` and Hive `STRUCT`-typed columns. ({issue}`15017`)
```

i.e. clarify that these are Hive types, not Trino types. To Trino both appear as ROW.